app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split("Command is wrong!")
```

Comment:
why did u do that? it doesn't make any sense. pls read how `split` works and add  `print("Command is wrong!") `after 4 line